tests/test-extensions/some-extension/src/main/java/org/mule/test/some/extension/SomeOps.java

```diff
@@ -44,11 +44,18 @@ public void someOp(@Connection String conn, @Config SomeExtension ext) {}
    * <it>isOneRequired</it> property.
    *
    * @param oneParameterGroup some test operation config with a isOneRequired exclusive-optional configuration
-   * @return one of the configs arguments
+   * @return the config
    */
-  public String oneRequiredParameterResolverOperation(@ParameterGroup(
+  public SomeParameterGroupOneRequiredConfig oneRequiredParameterResolverOperationDslTrue(@ParameterGroup(
```

Comment:
Any particular reason not to create a new one instead of changing the existing one?

Reply:
Just for better clarity and readability. Also to avoid repetition.

Reply:
This also means that every usage of this operation probably needs to change. I would create a new one.